### ISSUE TITLE
Break sequence?

### ISSUE THREAD

**Issue by richardhkirkando** [beagleterm/beagle-term#45]
Is it possible to issue a break sequence from beagle term?  Obviously, no break key on most chromebooks.

**Comment by sunglim** [comment]
For now, "No".

And it's impossible now, because `chrome.serial.setBreak` is committed on m45 (latest chrome stable is m44)

**Comment by RandomlyKnighted** [comment]
The latest version of Chrome is now version 52 so [this is now possible](https://developer.chrome.com/apps/serial#method-setBreak).

**Comment by sunglim** [comment]
@RandomlyKnighted  that's true. I implemented [that feature](https://bugs.chromium.org/p/chromium/issues/detail?id=461169) on chrome side :) 

Now we can implement this issue.  PR is always welcome.
